src/service/fixed-layer.js

```diff
@@ -158,12 +157,19 @@ export class FixedLayer {
 
   /**
    * Must be always called after DOMReady.
+   * @return {boolean}
    */
   setup() {
+    const viewer = Services.viewerForDoc(this.ampdoc);
```

Comment:
Worth considering if this should return a boolean, especially since its only needed for tests.